src/get-replacement.js

```diff
@@ -24,6 +24,8 @@ function stringToAST(string) {
     plugins: [
       // add more on request...
       'jsx',
+      'dynamicImport',
+      'objectRestSpread',
```

Comment:
Maybe it makes sense to add more? What are the implications?
See: https://github.com/babel/babel/tree/master/packages/babel-parser#plugins